README.md

```diff
@@ -1,68 +1,84 @@
 # React Native Zapp Bridge
 
-A React Native package that include bridging from React Native to [Applicaster](http://www.applicaster.com) Zapp Platform native framework.
+A React Native package that include bridging from React Native to
+[Applicaster](http://www.applicaster.com) Zapp Platform native framework.
 
 ## Features
 
 React-Native-Zapp-Bridge is a util package that enable you to:
 
-1. **Analytics** - use `sendAnalyticEvent` to send an analytics event throw [Zapp Morpheus](http://developer-zapp.applicaster.com/analytics/morpheus/morpheus.html) (Zapp Analytics Manager). 
+1. **Analytics** - use `sendAnalyticEvent` to send an analytics event throw
+   [Zapp Morpheus](http://developer-zapp.applicaster.com/analytics/morpheus/morpheus.html)
+   (Zapp Analytics Manager).
 
-2. **Video Player** - use `APVideoPlayer` to add a video player to your React-Native screen, this will add the chosen pluggable player on the Zapp Platform.
+2. **Video Player** - use `APVideoPlayer` to add a video player to your
+   React-Native screen, this will add the chosen pluggable player on the Zapp
+   Platform.
 
 ## Installation
 
-`$ npm install --save react-native-zapp-bridge `
-
+`$ npm install --save react-native-zapp-bridge`
 
 ## Usage
 
 ### Analytics
-  1. import:
-  ```javascript
-  import { sendAnalyticEvent } from 'react-native-zapp-bridge';
-  ```
-  2. send event:
-  ```javascript
-  sendAnalyticEvent(`Your Screen Name`, {
-      param: value,
-    })
-      .then(console.log)
-      .catch(console.warn);
-  }
-  ```
-  
+
+1. import:
+
+```javascript
+import { sendAnalyticEvent } from 'react-native-zapp-bridge';
+```
+
+2. send event:
+
+```javascript
```

Comment:
whats the change in the sendAnalyticsEvent?

Reply:
I have a code formatting on save enabled...